nestjs/src/gratitudes/gratitudes.controller.ts

```diff
@@ -14,8 +14,7 @@ export class GratitudesController {
   @ApiOperation({ operationId: 'createGratitude' })
   @ApiOkResponse({ type: GratitudeDto })
   public async create(@Req() req: CurrentRequest, @Body() dto: CreateGratitudeDto) {
-    const data = await this.service.create(req.user, dto);
-    return new GratitudeDto(data);
+    await this.service.create(req.user, dto);
```

Comment:
currently it is responding with 201 status code for successful requests - not sure if we should return all the successful feedbacks, let me know if we should be returning something from this controller

Reply:
it's fine